### PR TITLE
feat : Added login input

### PR DESCRIPTION
**Pull request by sudipkundu999**
Added login input container to be used in future projects.

File structure:
components>input>input.css => contains CSS for input only
docs>component-presentation>input-presentation.css => contains CSS for presenting the input on the documentation site.

### CODE REVIEW

File: Components/input/input.css [hunk 1]
```diff
@@ -29,3 +29,69 @@
...
+    flex-direction: column;
+    align-items: center;
+    border-radius: 10px;
+    overflow: hidden;
+    padding: 3rem 1rem;
+}
+  
+
+.login-container {
+    background-color: #ffffff;
```
Comment (archit7448) [inline Components/input/input.css]: use var

File: Components/input/input.css [hunk 2]
```diff
@@ -29,3 +29,69 @@
     height: 50%;
     padding: 1rem;
 }
+
+/* Login container starts here */
+.login-container {
+    background-color: #ffffff;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
```
Comment (archit7448) [inline Components/input/input.css:36]: use var

**Comment by archit7448** [comment]
looks good to me but only these changes

**Review by archit7448 — APPROVED** [review]

**Review by DivyanshSareen — APPROVED** [review]
Looks good. Proper use of labels 💯. Do the changes and you are good to merge

File: Components/input/input.css (hunk 2 above)
Comment (sudipkundu999) [reply Components/input/input.css:36]: this is intentional as the color is not repeating itself

**Pull request merged**